src/Microsoft.AspNet.Mvc.ModelBinding/Binders/FormCollectionModelBinder.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    /// <summary>
+    /// Modelbinder to bind form values to <see cref="IFormCollection"/>.
+    /// </summary>
+    public class FormCollectionModelBinder : IModelBinder
+    {
+        /// <inheritdoc />
+        public async Task<bool> BindModelAsync([NotNull] ModelBindingContext bindingContext)
+        {
+            if (!typeof(IFormCollection).GetTypeInfo().IsAssignableFrom(
```

Comment:
This wouldn't work with something like: `class MyFormCollection : FormCollection { }`

I expect it to not bind, but it will probably throw